grails-app/domain/au/org/ala/doi/Doi.groovy

```diff
@@ -65,7 +64,6 @@ class Doi {
 
     static mapping = {
         doi type: CitextType
-        provider defaultValue: DoiProvider.byName(Holders.config.doi.service.provider as String)
```

Comment:
The `defaultValue` is only used when building the database schema. The doi-service uses flyway to manage the DB schema so `defaultValue` is redundant.